terraform/groups/configurable-api-caller/main.tf

```
@@ -252,6 +252,34 @@ resource "aws_lambda_permission" "allow_cloudwatch_efs_submit_files_to_fes" {
   source_arn    = aws_cloudwatch_event_rule.efs_submit_files_to_fes.arn
 }
 
+//
+// Set to only deploy on development. As service is only deployed on development.
+// Future work required to double check staging and live profiles are still correct, when count is changed.
+//
+resource "aws_cloudwatch_event_rule" "call_api_caller_lambda_account_validator_cleanup_submissions" {
+  count               = var.deploy_to == "development" ? 1 : 0 
+  name                = "call_api_caller_lambda_account_validator_cleanup_submissions"
+  description         = "Cloudwatch event to call ${aws_lambda_function.configurable_api_lambda.function_name} lambda at 2am everyday"
+  schedule_expression = "cron(0 2 * * ? *)"
+}
+
+resource "aws_cloudwatch_event_target" "event_target_api_caller_account_validator_cleanup_submissions" {
+  count     = var.deploy_to == "development" ? 1 : 0
+  target_id = aws_cloudwatch_event_rule.call_api_caller_lambda_account_validator_cleanup_submissions[0].id
+  rule      = aws_cloudwatch_event_rule.call_api_caller_lambda_account_validator_cleanup_submissions[0].name
+  arn       = aws_lambda_function.configurable_api_lambda.arn
+  input     = file("profiles/${var.aws_profile}/common-${var.aws_region}/account_validator_cleanup_submissions.json")
+}
+
+resource "aws_lambda_permission" "allow_cloudwatch_account_validator_cleanup_submissions" {
+  count         = var.deploy_to == "development" ? 1 : 0
```

Since this will be needed in all environments th count shouldn't be conditional.
It shoudl just be set to 1.
```suggestion
  count     =  1
```